stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/utils/ChannelUtils.kt

```diff
@@ -0,0 +1,6 @@
+package io.getstream.chat.android.offline.utils
+
+import io.getstream.chat.android.offline.experimental.global.GlobalMutableState
+
+internal fun isChannelMutedForCurrentUser(cid: String): Boolean =
+    GlobalMutableState.getOrCreate().channelMutes.value.any { mutedChannel -> mutedChannel.channel.cid == cid }
```

Comment:
I didn't have the full `Channel` at my disposal everywhere I needed to use this so I opted for a plain function with `cid: String` as argument.

Reply:
Can we add a KDoc here?

Reply:
Sure, I'll add one now.